src/live_restore/live_restore_server.c

```diff
@@ -55,7 +55,8 @@ __live_restore_clean_up(WT_SESSION_IMPL *session, WT_SESSION_IMPL *checkpoint_se
         uint64_t time_diff_ms;
         __wt_timer_evaluate_ms(session, &server->start_timer, &time_diff_ms);
         __wt_verbose(session, WT_VERB_LIVE_RESTORE_PROGRESS,
-          "Completed restoring %" PRIu64 " files in %" PRIu64 " seconds",
+          "Live restore background migration finished restoring %" PRIu64 " files in %" PRIu64
+          " seconds. Live restore will now begin cleanup",
```

Comment:
Could probably use a fullstop at the end of the second sentence.

Reply:
2569de11135cda3a925dc57e8ebe750892cac343